honeybee_schema/radiance/wea.py

```diff
@@ -0,0 +1,100 @@
+"""Wea Schema"""
+from pydantic import Field, constr, confloat
+from typing import List
+import datetime
+
+from .._base import NoExtraBaseModel
+
+
+class Location(NoExtraBaseModel):
+    """Used to specify latitude and longitude of a location on the Earth."""
+
+    type: constr(regex='^Location$') = 'Location'
+
+    latitude: float = Field(
+        ...,
+        ge=-90,
+        le=90,
+        description='Location latitude between -90 and 90.'
+    )
+
+    longitude: float = Field(
+        ...,
+        ge=-180,
+        le=180,
+        description='Location longitude between -180 (west) and 180 (east).'
+    )
+
+    time_zone: int = Field(
```

Comment:
This should be a float. Look up `timezone tehran` as an example.

Reply:
Ah, you are right. We also made it a float in case people want to set the time zone to be aligned with solar time. I'll fix it.